project_euler/problem_04/sol1.py

```diff
@@ -40,12 +38,8 @@ def solution(n):
                 if (number % divisor == 0) and (len(str(number // divisor)) == 3.0):
                     return number
                 divisor -= 1
-    raise ValueError("That number is larger than our acceptable range.")
+    return -1
```

Comment:
Um, why are you changing the code?
The main goal for this PR is just to fix type hints and default arguments. Please don't make it hard for us to review

Reply:
My mistake, I'll scope my changes down to the default arguments and type hints.